client-app/shared/account/components/sign-in-form.vue

```diff
@@ -99,8 +103,9 @@ const { errors, handleSubmit } = useForm({
 
 const { value: userName } = useField<string>("userName");
 const { value: password } = useField<string>("password");
+const rememberMe = ref(true);
```

Comment:
Maybe it shouldn't be active by default. Need to check/discuss with BA/PO.

Reply:
@Flanker72 Fixed